algorithmic_efficiency/workloads/imagenet_resnet/imagenet_jax/randaugment.py

```diff
@@ -8,7 +8,8 @@
 import math
 
 import tensorflow as tf
-from tensorflow_addons import image as contrib_image
+
+#from tensorflow_addons import image as contrib_image
```

Comment:
Hello @runame , this is because tensorflow_addons is not supported now, see this : https://www.tensorflow.org/addons/tutorials/image_ops

Also, the current TF version is 2.18 but this requires < 2.15, we might have to re-write the functions used in the current code for rotate, translate, shear operations or use TF 2.15

Reply:
Thanks for the explanation. Do I understand correctly that the code is not runnable then (on this branch)?

Reply:
Yes, the code wont run incase you want to use the functions from tensorflow_addons, I think the CI checks didnt require it so it passed the checks and other tests, we have to write a custom logic or use another package for the same, shall we stick to TF2.15 then? but then the numpy will also change I think it needs numpy <2

Reply:
Maybe we should upgrade TF in a separate PR that also adjusts the code to work and be functionally equivalent to the previous one?